recipes/dotnet-interactive/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "dotnet-interactive" %}
+{% set version = "1.0.157409" %}
+{% set commit = "6dfc99911743a633b5edbac09a8c5fe5e5e9f980" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dotnet/interactive/archive/{{ commit }}.zip
+  sha256: cc2de6715bc7c977b3ab7bf95b8ef12ce63494ac0e29f48a6eef06a8dc75596b
+
+build:
+  skip: true  # [not win]
+  number: 0
+
+requirements:
+  build:
+    - dotnet >=5.0.100
+  run:
+    - dotnet >=5.0.100
```

Comment:
Is this really just a lower bound or should there also be an upper limit? i.e. is a hypothetical dotnet 6 or dotnet 15 expected to work just fine as well?

Reply:
That's an interesting question! It seems minor versions *should* be compatible:
https://docs.microsoft.com/en-us/dotnet/core/versions/selection#framework-dependent-apps-roll-forward

![image](https://user-images.githubusercontent.com/881019/100725497-d5bcb500-340f-11eb-894a-eedd34628ed6.png)

...I'll optimistically pin to <6.0.